server-lib/Mux.js

```diff
@@ -1,8 +1,41 @@
+import { cache } from 'react'
 import Mux from '@mux/mux-node'
 
 // Assumes you have your access token set in environment variables:
 // Access Token ID: process.env.MUX_TOKEN_ID
 // Access Token Secret: process.env.MUX_TOKEN_SECRET
 const { Video } = new Mux()
 
-export { Video }
+const listAssets = cache(async () => {
+  try {
+    const assets = await Video.Assets.list()
+    return assets
+  } catch (e) {
+    console.error(e)
+  }
+})
+
+const getAssetByPlaybackId = cache(async (playbackId) => {
+  // find asset by playback id from list and return
+  const assets = await listAssets()
+
+  const asset = assets.find((asset) => {
+    asset.playback_ids.some((pid) => {
+      return pid.id === playbackId
+    })
+  })
+
+  return asset
+})
+
+const getAsset = cache(async (assetId) => {
+  // find asset from list and return
+  const assets = await listAssets()
+  const asset = assets.find((asset) => {
+    return asset.id === assetId
+  })
+
+  return asset
+})
```

Comment:
Should we use Mux APIs here? Or reference the cached list?

Referencing the cached list is probably more efficient because we don't need to hit the Mux API multiple times.
Using Mux APIs is... less fancy and maybe easier to understand.

Reply:
I'd prefer we use the Mux API for this. It's what I would do in a real app. https://muxinc.github.io/mux-node-sdk/class/src/video/resources/assets.js~Assets.html#instance-method-get